recipes/botan/all/conanfile.py

```diff
@@ -0,0 +1,310 @@
+from conans.model.version import Version
+
+
+class BotanConan(ConanFile):
+    name = 'botan'
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/randombit/botan"
+    license = "BSD-2-Clause"
+    exports = ["patches/*"]
+    description = "Botan is a cryptography library written in C++11."
+    settings = 'os', 'arch', 'compiler', 'build_type'
+    options = {
+        'amalgamation': [True, False],
+        'bzip2': [True, False],
+        'debug_info': [True, False],
+        'openssl': [True, False],
+        'quiet': [True, False],
+        'shared': [True, False],
+        'fPIC': [True, False],
+        'single_amalgamation': [True, False],
+        'sqlite3': [True, False],
+        'zlib': [True, False],
+        'boost': [True, False],
+        'enable_modules': "ANY",
+        'system_cert_bundle': "ANY"
+    }
+    default_options = {'amalgamation': True,
+                       'bzip2': False,
+                       'debug_info': False,
+                       'openssl': False,
+                       'quiet': True,
+                       'shared': True,
+                       'fPIC': True,
+                       'single_amalgamation': False,
+                       'sqlite3': False,
+                       'zlib': False,
+                       'boost': False,
+                       'enable_modules': None,
+                       'system_cert_bundle': None}
+
+    def configure(self):
+        compiler = self.settings.compiler
+        version = Version(self.settings.compiler.version.value)
+
+        if self.settings.os == "Windows" and compiler == "Visual Studio" and version < "14":
+            raise ConanInvalidConfiguration("Botan doesn't support MSVC < 14")
+
+        elif compiler == "gcc" and version >= "5" and compiler.libcxx != "libstdc++11":
+            raise ConanInvalidConfiguration(
+                'Using Botan with GCC >= 5 on Linux requires "compiler.libcxx=libstdc++11"')
+
+        elif compiler == "clang" and compiler.libcxx not in ["libstdc++11", "libc++"]:
+            raise ConanInvalidConfiguration(
+                'Using Botan with Clang on Linux requires either "compiler.libcxx=libstdc++11" ' \
+                'or "compiler.libcxx=libc++"')
+
+        if self.options.boost:
+            self.options["boost"].add("shared=False")
+            self.options["boost"].add("magic_autolink=False")
+            self.options["boost"].add("without_coroutine=False")
+            self.options["boost"].add("without_system=False")
+
+    def requirements(self):
+        if self.options.bzip2:
+            self.requires("bzip2/1.0.6")
+        if self.options.openssl:
+            self.requires("openssl/1.0.2t")
+        if self.options.zlib:
+            self.requires("zlib/1.2.11")
+        if self.options.sqlite3:
+            self.requires("sqlite3/3.30.1")
+        if self.options.boost:
+            self.requires("boost/1.70.0")
```

Comment:
```suggestion
            self.requires("boost/1.71.0")
```
I would say you should try 1.71 if possible

Reply:
FWIW: Both this recipe and our production application (using Botan's boost integration) work with `boost/1.71`. @hrantzsch

Reply:
Sorry, I somehow missed this comment